src/main/java/jdi/forms/MetalsColorsForm.java

```diff
@@ -0,0 +1,51 @@
+package jdi.forms;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import jdi.objects.MetalsColors;
+
+public class MetalsColorsForm extends Form<MetalsColorsForm> implements MetalsColorsInt {
```

Comment:
No need to create your own interface.
In JDI LIght exist class Form<T>, that your class extends. 
This class has a fill method that needs to be overridden.
![image](https://user-images.githubusercontent.com/74856431/215040897-6f92bb8d-19ab-4023-b8dc-6aee9e2230d6.png)